artemis-quorum-ri/pom.xml

```diff
@@ -43,6 +47,12 @@
       <dependency>
          <groupId>org.apache.zookeeper</groupId>
          <artifactId>zookeeper</artifactId>
+         <scope>test</scope>
```

Comment:
Also not sure about this one changing to test. The main purpose of this module is currently to use ZK. That it does it via the curator stuff doesnt really mean this is a test-only dep, whilst listing it like this really makes it seem like it is...however anything using this module will still get ZK since it isnt really. Maven doesnt have a way of saying "runtime + test" which is what this really is. I think I'd leave it at compile for clarity.